lib/ansible/module_utils/vmware.py

```diff
@@ -319,24 +491,36 @@ def connect_to_api(module, disconnect_atexit=True):
     validate_certs = module.params['validate_certs']
 
     if validate_certs and not hasattr(ssl, 'SSLContext'):
-        module.fail_json(msg='pyVim does not support changing verification mode with python < 2.7.9. Either update '
-                             'python or or use validate_certs=false')
+        msg = 'pyVim does not support changing verification mode with'
+        msg += ' python < 2.7.9. Either update python or or use'
+        msg += ' validate_certs=false'
+        module.fail_json(msg=msg)
```

Comment:
This should not be necessary for PEP8. We have a line-length of 160, not the default 80.
I really prefer the former over the latter.